===================================================================
RCS file: /cvs/gnome/totem/src/backend/bacon-video-widget-xine.c,v
retrieving revision 1.167
retrieving revision 1.168
diff -u -r1.167 -r1.168
--- bacon-video-widget-xine.c	2005/01/30 22:27:50	1.167
+++ bacon-video-widget-xine.c	2005/02/07 08:37:00	1.168
@@ -2447,6 +2447,10 @@
 {
 	xine_cfg_entry_t entry;
 
+	g_return_if_fail (bvw != NULL);
+	g_return_if_fail (BACON_IS_VIDEO_WIDGET (bvw));
+	g_return_if_fail (path != NULL);
+
 	/* DVD device */
 	bvw_config_helper_string (bvw->priv->xine, "media.dvd.device",
 			path, &entry);
